apps/ideaspace/src/components/common/SignInButton/SignInButton.js

```diff
@@ -3,7 +3,7 @@ import {
   MenuButton
 } from "./StyledSignInButton";
 
-export default function SignInButton({redirectUrl=(process.env.NEXT_PUBLIC_FRONT_END_URL + "/users/me")}) {
+export default function SignInButton({redirectUrl="https://devlaunchers.org/users/me"}) {
```

Comment:
is this change necessary? it will prevent developers and testers from having the expected sign in experience in staging/local environments.